operator/test/test.sh

```diff
@@ -4,7 +4,6 @@
 set -o errexit
 set -o nounset
 set -o pipefail
-set -x
```

Comment:
OK so I see where you turned off debug by default, but I don't see where the `--debug` arg is getting processed ... is that a default bash thing I'm missing ?

Reply:
This done in the parse_args function, around line 48/49.

Reply:
ah cool leveraging an existing arg whose impl was too far from the deltas of this PR for me to see ... thanks for the ptr